cmake/configs/nuttx_px4fmu-v2_common.cmake

```diff
@@ -187,6 +181,12 @@ set(config_module_list
 	#examples/hwtest
 )
 
+set(config_module_list
+	${general_modules_section1}
+	${estimation_modules}
+	${general_modules_section2}
+)
+
```

Comment:
Without this I was still getting increased mem usage even after parm change, so I think it is best to leave it as is.